Mimir/src/helpers/onlineLog/Parser.php

```diff
@@ -50,6 +50,8 @@ class OnlineParser
      */
     protected $_lastTokenIsAgari = false;
 
+    protected $_ankan_cache = [];
```

Comment:
Please make sure to run `make check` locally to run all code style tools, linters and tests. Absence of phpdoc comment for class member should output static analysis error.

Reply:
fixed